### DISCUSSION TITLE
is lucid support mongoDB

### DISCUSSION THREAD

**Discussion by akirawinz** [adonisjs/core#1752] [Help] [answered]
as I read in document , I didn't see any information about mongoDB

after I use ` node ace invoke @adonisjs/lucid` command  their is no mongoDB too. Is their anyway to use Mongo ?

**Comment by thetutlage** [comment, marked as answer]
Use any MongoDB ORM/library as you would do with any other Node.js application. `@adonisjs/lucid` is SQL only

**Reply by shubham0503** [reply]
Hi @thetutlage,

As you mentioned that we can use a separate library for MongoDB. So can you please tell me? how I can make use of migrations and seeder while using the separate library(mongoose) for MongoDB.

**Reply by aayush123** [reply]
@shubham0503 Migrations and seeders are ORM specific features. When using a different ORM than Lucid, you'll have to use migrations and seeders provided by that library, if they are available.

**Reply by RomainLanz** [reply]
> As you mentioned that we can use a separate library for MongoDB. So can you please tell me? how I can make use of migrations and seeder while using the separate library(mongoose) for MongoDB.

If you want to use something else than Lucid, you'll loose all Lucid's benefits. This includes migrations, seeders, query builder, relation mapping, auth provider, etc.
